package.json

```diff
@@ -37,6 +36,7 @@
     "tmp": "^0.2.1",
     "urllib": "^2.38.0",
     "uuid": "^3.3.2",
+    "vm": "0.1.0",
```

Comment:
Do we need to add this library? It was released 7 years ago https://www.npmjs.com/package/vm?activeTab=readme and it has information that we should not install it since it's part of node now https://github.com/DiegoRBaquero/node-vm
(I removed this dependency and it seems to work fine)

Reply:
It's removed now